srtcore/core.cpp

```diff
@@ -2668,6 +2668,82 @@ int CUDT::processSrtMsg_HSRSP(const uint32_t *srtdata, size_t len, uint32_t ts,
     return SRT_CMD_NONE;
 }
 
+void CUDT::interpretRejectionMessage(const CHandShake& hs, const CPacket& hspkt)
+{
+    // This is simply part of the same procedure as done in interpretRejectionMessage(),
```

Comment:
search-replace was too good. It should be "as done in `interpretSrtHandshake()`".